src/components/Sidebar/ScenePane.jsx

```diff
@@ -34,6 +38,24 @@ function ScenePane({ closeCallback }) {
   const propertyOwners = useSelector((state) => state.propertyTree.propertyOwners, shallowEqual);
   const propertyOwnersScene = propertyOwners.Scene?.subowners ?? [];
 
+  // Add featured/interesting nodes in a separate list
+  const interestingNodes = [];
+  propertyOwnersScene.forEach((uri) => {
+    if (hasInterestingTag(uri, propertyOwners)) {
+      interestingNodes.push({
+        key: uri,
+        uri,
+        expansionIdentifier: `scene-search/${uri}`
+      });
+    }
+  });
+  // For now, sort alphabetically. Later we might want to keep the order in the profile
```

Comment:
Why do we want to sort instead of keeping the order? Isn't preserving the order the desired functionality?

Reply:
Yes, but the way the nodes are collected does currently not preserve the order in which they are specified in the profile.  It is "random" (probably based on the order the assets are added)

We need to send the ordered list ot the UI somehow, e.g. in a topic with the information from the profile. Hence the comment here (Should maybe even be a @TODO comment)